hack/annotate.sh

```diff
@@ -25,6 +26,10 @@ retry() {
       sleep $delay
     else
       echo "Retry $count/$retries exited $rc, no more retries left." > /dev/stderr
+	  if [ $keep_going -eq 1 ]; then
+	  	echo '$keep_going is set, failure is going to be skipped.' > /dev/stderr
+	  	return 0
+	  fi
```

Comment:
I think moving on with the job is a good idea, but ultimately the jobs still needs to fail if there is an error on any attempt.